Adds contribution guidelines and a code of conduct.

jessex opened the pull request:
This handles #20. It copies the guidelines and code of conduct from recidiviz-data.

jessex commented:
@wmginsberg, @matthewroot: would appreciate approval on this one. Thanks!

matthewroot commented:
Will try to take a look today.
On Fri, May 4, 2018 at 12:36 AM Joshua Essex <notifications@github.com>
wrote:

> @wmginsberg <https://github.com/wmginsberg>, @matthewroot
> <https://github.com/matthewroot>: would appreciate approval on this one.
> Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Recidiviz/recidiviz.com/pull/21#issuecomment-386526208>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABdeEbCNTQzH7nK72qAtA2xXKmm3eQ6Iks5tvAT5gaJpZM4Tu6-m>
> .
>

matthewroot commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,150 @@
+
+## Style
+
+Follow the [Airbnb Javascript Style Guide](https://github.com/airbnb/javascript).
```
Follow the Airbnb Style Guides for [Javascript](https://github.com/airbnb/javascript) and [React](https://github.com/airbnb/javascript/tree/master/react).

matthewroot commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,150 @@
+
+**tl;dr:**
+1. Work on a feature branch
+1. Run linting and tests often
```
Link to README.md section would be nice here.

matthewroot commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,150 @@
+of the repository.
+
+Before issuing a pull request, run our linting and test suite over your changes
+to ensure no regression, as described in [our Readme](README.md).
```
Linking to specific section would be good here as well.

matthewroot commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,150 @@
+
+The linting configuration should adhere to this style guide and help you follow
+it automatically. You can use support for warning suppression if you find it
+necessary, but this may be argued against during code review.
```
May also be worth linking to the section in the README on application organization as that's a Style-ish thing.

jessex commented:
@matthewroot: I've addressed all feedback. And yes, this commit incorporated all changes from the recidiviz-data PR already. Ready for final approval.
